libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -386,6 +386,14 @@ present in the chain during the handshake, it will be added to the
 `certificate_authorities` list and the handshake will continue
 normaly.
 
+One way to get the fingerprint from a CA certificate is using the
+following command where `ca.crt` is the certificate.
```

Comment:
You may want to explicitly state that this applies to Unix like systems (Linux, Mac). It is obvious to us but users who only know Windows or are not very technical may read this as well.

Reply:
Done on [b63382e](https://github.com/elastic/beats/pull/33507/commits/b63382e966a7b8c11da018cad97e47ebde72d35c).